Header header

#Position of object in frame
int16 pos_x
int16 pos_y

#Camera frame size
int16 frame_width
int16 frame_height

#range [0 , 1]
float64 confidence
